types/react-visibility-sensor/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for react-visibility-sensor 3.11
+// Project: https://github.com/joshwnj/react-visibility-sensor#readme
+// Definitions by: Rasmus Bergström <https://github.com/JRasmusBm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+export interface Shape {
+  top?: number;
+  left?: number;
+  bottom?: number;
+  right?: number;
+}
+
+interface Props {
+  onChange: (isVisible: boolean, visibilityRect?: Shape) => void;
+  active?: boolean;
+  partialVisibility?: boolean;
+  offset?: Shape;
+  minTopValue?: number;
+  intervalCheck?: boolean;
+  intervalDelay?: number;
+  scrollCheck?: boolean;
+  scrollDelay?: number;
+  scrollThrottle?: number;
+  resizeCheck?: boolean;
+  resizeDelay?: number;
+  resizeThrottle?: number;
+  delayedCall?: boolean;
+  children?: (
+    args: { isVisible: boolean; visibilityRect?: Shape }
+  ) => React.ReactNode;
+}
+
+declare const ReactVisibilitySensor: React.StatelessComponent<Props>;
+
+export default ReactVisibilitySensor;
```

Comment:
It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).

Reply:
Thank you for the time to review my pull request. Is this how you meant?